pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -445,11 +447,13 @@ func (ctx *context) sendOutboundAck(msg dispatcher.DIDCommMsg) error {
 
 	err := json.Unmarshal(msg.Payload, ack)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	// TODO : Issue-353
 	sendVerKey := temp
-	return ctx.outboundDispatcher.Send(ack, sendVerKey, destination)
+	return func() error {
```

Comment:
Suggest assigning the func to a variable as a new line. (and then returning that variable.)

Reply:
done